07-the-model-workflow.Rmd

```diff
@@ -326,9 +326,9 @@ ames_test  <-  testing(ames_split)
 lm_model <- linear_reg() %>% set_engine("lm")
 
 lm_wflow <- 
-  workflow() %>% 
+  lm_wflow %>% 
```

Comment:
This should still be just plain `workflow()`, right? We don't define `lm_wflow` in this section at all.